### ISSUE TITLE
failed to read ARW from supported sony ILCE a6000, darktable 4.0

### ISSUE THREAD

**Issue by drgnkpx** [darktable-org/darktable#12371]
First time installing darktable, can't open any of my ARW files from a sony a6000 camera, failed to read white balance information

* darktable version : 4.0.0
* OS   :           win11 64bit
* Memory            : 64gb
* Graphics card     : RTX 2070 super, GTX 980
* Graphics driver   : 516.94

**Comment by Mark-64** [comment]
DT 4.0 on Win 11, Nvidia 1060, I can read A600 ARW files without issues. Did you do a clean install of DT?

**Comment by drgnkpx** [comment]
Of course it's a clean install, never installed it before lol

**Comment by Nilvus** [comment]
So, it's probably a duplicate of issue #11062.Could your camera be a "tweaked" A6001? If not, provide a RAW that give you this error.

**Comment by drgnkpx** [comment]
Ah, that would be it yeah, tweaking it is the only way to enable remote video capture support in the mobile app. Just waiting on an alias then?

**Comment by kmilos** [comment]
Feel free to add it yourself to your cameras.xml, there's plenty of alias examples in there.

**Comment by drgnkpx** [comment]
Yep, that did it, thanks for the help ^^

**Comment by Nilvus** [comment]
So I close.
